src/main/java/com/project/daechiwon/domain/user/type/UserType.java

```diff
@@ -1,5 +1,17 @@
 package com.project.daechiwon.domain.user.type;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+
 public enum UserType {
     NORMAL, TEACHER
+
+    @JsonCreator
+    public static UserType fromTestEnum(String val){
```

Comment:
Hey! This may return null.. ㅡㅡ
Make this code to use `Optional`, and add the logic that return `400 Bad request` when result of `isEmpty()` is true..